airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -50,5 +50,11 @@
     "required": ["account_id", "access_token", "start_date"]
   },
   "supportsIncremental": true,
-  "supported_destination_sync_modes": ["append"]
+  "supported_destination_sync_modes": ["append"],
+  "authSpecification": {
+    "auth_type": "oauth2.0",
+    "oauth2Specification": {
+      "oauthFlowInitParameters": [["access_token"]]
```

Comment:
this should be updated as a output parameter using the new spec

Reply:
@sherifnada Done, could u review this PR one more time?